### PR TITLE
Use IPFS 52 for serviceLauncher

### PR DESCRIPTION
**Pull request by valmack**
So that the IPFS version we are using in the `local_node-internal` test is compatible with dev infra as of right now

### CODE REVIEW

**Review by ukstv — APPROVED** [review]

File: build_submodules.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+cd ./src/submodules/js-ceramic
+npm install
+cd ./packages/ipfs-daemon
+npm install
```
Comment (stbrody) [inline build_submodules.sh:5]: I don't think this second npm install should be necessary.  Also not sure if it's harmful but maybe could be?  idk lerna is weird

File: src/serviceLauncher.ts [hunk 2]
```diff
@@ -25,13 +25,13 @@ function main() {
...
         if (service.type == 'ipfs') {
-            subprocess = exec(`node node_modules/@ceramicnetwork/ipfs-daemon/bin/ipfs-daemon`,
+            subprocess = exec(`node src/submodules/js-ceramic/packages/ipfs-daemon/bin/ipfs-daemon`,
                 {
                     env: {
                         ...process.env,
                         IPFS_API_PORT: service.port,
                         CERAMIC_NETWORK: 'dev-unstable',
-                        DEBUG: 'bitswap*'
+                        DEBUG: '*error,bitswap*,ipfs:repo,libp2p:connection-manager,libp2p,libp2p:transports,libp2p:tcp*,libp2p:gossipsub*,libp2p:persisten-peer-store,libp2p:peer-store*,libp2p:websockets,libp2p:dialer'
```
Comment (stbrody) [inline src/serviceLauncher.ts:34]: Do we want to add all this extra logging at the same time?  Seems like we should keep the change focused on just downgrading IPFS

Comment (valmack) [reply src/serviceLauncher.ts:34]: Fair

File: build_submodules.sh (hunk 1 above)
Comment (valmack) [reply build_submodules.sh]: 👍

**Review by stbrody — APPROVED** [review]
LGTM though hopefully we won't ever have to merge this

**Comment by valmack** [comment]
Closing without merge because we don't need this now that tests are passing on the newer IPFS version

**Pull request closed without merging**